tests/smoke/Commands/dev/geb.smoke.yaml

```diff
@@ -5,23 +5,23 @@ tests:
     command:
       - juvix
       - --no-colors
-      - dev 
+      - dev
       - geb
       - repl
     stdout:
-      contains: "Welcome to the Juvix Geb REPL!"
+      contains: 'Welcome to the Juvix Geb REPL!'
     exit-status: 0
-    
+
```

Comment:
We should stop ignoring in the pre-commit task any file in tests/smoke. Atm, we ignore the whole `tests` folder.